federated_learning/financial_fraud_detection/README.md

```diff
@@ -80,6 +87,21 @@ cd tree-based/
 nvflare simulator jobs/creditcard_5_bagging_IID_split_uniform_lr -w ${PWD}/workspaces/xgboost_workspace_5_bagging_exponential_split_scaled_lr -n 5 -t 5
 ```
 
+You will see the execution as shown in the picture
+![execution](./figures/execution.png)
+Each Client loads the global model and does the local training, gets the train-auc and valid-auc.
+After the federated learning phase, the server will store the global model.
+![execution](./figures/store_model.png)
+
+#### Step 4. Prediction by the global model
+
+This is a simple example that utilizes a specific piece of data from the dataset to confirm that the prediction result is consistent with the ground truth. Additionally, you have the flexibility to independently modify the input for predictions.
```

Comment:
I don't understand what "independently modify" means. Can you provide more detailed or an example?

Reply:
This tells users that they can modify the prediction data and input them into the global model according to their own needs.